DonoGear/app/src/main/java/com/example/donogear/actionpages/MainActivity.java

```diff
@@ -383,6 +443,32 @@ public void onTabReselected(TabLayout.Tab tab) {
         });
     }
 
+    private void manageInnerHomeTabs() {
+        innerHomeTabs = findViewById(R.id.innerHometabs);
+
+        // Load the tab at index 0 which is Donor
+        innerHomeTabs.getTabAt(0).select();
+        innerBrowseTabs.addOnTabSelectedListener(new TabLayout.OnTabSelectedListener() {
+            @Override
+            public void onTabSelected(TabLayout.Tab tab) {
+
+                switch (tab.getPosition()) {
+                    case 0:
+                        loadFragment(new BrowsePageFragment(), HOME_IDENTIFIER);
+                        break;
+                }
+            }
+
+            @Override
+            public void onTabUnselected(TabLayout.Tab tab) {
```

Comment:
If these 2 methods aren't being used, they should be removed. Either that or add a TODO to modify them later